backend/copr_backend/storage.py

```diff
@@ -383,13 +383,18 @@ def delete_builds(self, dirname, chroot_builddirs, build_ids):
                 self.log.error("%s chroot path doesn't exist", chroot_path)
                 result = False
                 continue
+
             repository = self._get_repository(chroot)
             # Find the RPMs by list of build ids
             content_response = self.client.get_content(build_ids)
             list_of_prns = [package["prn"] for package in content_response.json()["results"] ]
             self.client.delete_content(repository, list_of_prns)
             self.log.info("Deleted resources: %s", list_of_prns)
 
+            # Delete results in the backend storage (logs, info files, etc)
```

Comment:
hehe, this is highly confusing because we have `BackendStorage` class, too (perhaps we could at some point rename it to `FilesystemStorage`, or alike).